scripts/ci/allFilesChangedInPatterns.mjs

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+import {setOutput} from '@actions/core';
+import {minimatch} from 'minimatch';
+import {getBaseHeadSHAs, getChangedFiles} from './git-utils.mjs';
+
+function getOutputName() {
+  return process.argv[2];
+}
+
+function getPatterns() {
+  return process.argv.slice(3);
+}
+
+function everyFileMatchOnePattern(files, patterns) {
```

Comment:
```suggestion
function everyFileMatchesAtLeastOnePattern(files, patterns) {
```

Reply:
@fbeaudoincoveo updated the method and the file name too! thanks!